addons/mail/static/src/models/emoji_registry.js

```diff
@@ -33,8 +33,8 @@ registerModel({
                 codepoints: emojiData.codepoints,
                 shortcodes: emojiData.shortcodes,
                 emoticons: emojiData.emoticons,
-                name: emojiData.name,
-                keywords: emojiData.keywords,
+                name: this.env._t(emojiData.name),
+                keywords: emojiData.keywords.map(keyword => this.env._t(keyword)),
```

Comment:
`_t()` requires the string to be in plain. It doesn't work with variables.
So the wrapping of `_t` must be done much earlier.

Reply:
I did test it and it was working. As even if they are keywords they hold String values only. 

I also got into an issue where if we replace the translated keywords with the original, we won't be able to make any search based on the original (ex. 'search' and it is also a common search parameter). So, I added some fixup for it. So, please take a look at that. Thanks!

Reply:
It's working if we have the entries in `POT` file.
Problem here is that the strings are not automatically registered in the `POT` file.
The solution to this problem is to wrap `_t()` where the string are in plain, which is in each occurrences of translatable strings in `emoji_data.js`.